NetKAN/SigmaBinary-core.netkan

```diff
@@ -1,22 +1,26 @@
 {
-    "spec_version": "v1.4",
-    "license": "CC-BY-NC-SA-4.0",
-    "$kref": "#/ckan/github/Sigma88/Sigma-Binary",
-    "$vref" : "#/ckan/ksp-avc",
-    "identifier": "SigmaBinary-core",
-    "name"        : "Sigma Binary",
-    "abstract"    : "This mod enables the creation of binary systems but does not contain any binary systems itself.",
+    "spec_version" : "v1.4",
+    "identifier"   : "SigmaBinary-core",
+    "$kref"        : "#/ckan/github/Sigma88/Sigma-Binary",
+    "$vref"        : "#/ckan/ksp-avc",
+    "name"         : "Sigma Binary",
+    "abstract"     : "This mod will let you turn custom and/or stock bodies into binary systems, without the need to do all the calculations yourself.",
+    "license"      : "CC-BY-NC-SA-4.0",
+    "author"       : "Sigma88",
     "depends": [
         { "name": "ModuleManager" },
-        { "name": "Kopernicus"}
+        { "name": "Kopernicus "}
```

Comment:
`"Kopernicus "` - you put the whitespace before the quotation mark instead of after.